pkg/crc/machine/driver_darwin.go

```diff
@@ -47,10 +45,6 @@ func updateKernelArgs(vm *virtualMachine) error {
 	}
 	defer sshRunner.Close()
 
-	if err := sshRunner.WaitForConnectivity(context.Background(), 20*time.Second); err != nil {
```

Comment:
this was added because we observed some errors related to ssh connectivity during testing, see https://github.com/code-ready/crc/pull/3302/files#r945923907

Reply:
yes, it was added because of ssh connection delay happen sometime and observed by QE also iirc.

Reply:
Added questions to https://github.com/code-ready/crc/pull/3302/files#r955160463 to get a better understanding of why this was added, and rework this patch accordingly.

Reply:
I replied to the comments in the other PR, but could not reproduce it again with both 4.11.0 or 4.11.1 bundles

Reply:
Tried 3 times. and didn't get error so now I don't have any strong opinion and just see this wait as safe option only. (in case there is some ssh connection timeout so retry will work). I am OK both options (remove or have it with comment).